components/inputs/input-date.js

```diff
@@ -313,20 +316,25 @@ class InputDate extends LocalizeStaticMixin(LitElement) {
 	_handleDropdownClose() {
 		this._calendar.reset();
 		this._dropdownOpened = false;
+		this._textInput.scrollIntoView({block: 'nearest', behavior: 'smooth', inline: 'nearest'});
```

Comment:
Sometimes (I think mostly noticed on iPhone) when the dropdown closes the input-text does not get scrolled to, and it seems like it should give that it has focus.